source/assets/stylesheets/components/_cards.scss

```diff
@@ -0,0 +1,156 @@
+// Variables
+
+$light:  #febbbb;
+$medium: #fe9090;
+$dark:   #ff5c5c;
+$dark-v1: #c84848;
+$text-color: #999;
+$cta: #00c5a7;
+$card-font-small: 15px;
+
+
+// Global Styles
+/*#press {
+  background: $light;
+  background: linear-gradient(135deg, $light 0%,$medium 45%,$dark 100%);
+  font-size: 10px;
+  padding: 2em;
+}
+.full-screen {
+  height: 100vw;
+  position: relative;
+}*/
+
+
+// General content
+.title {
+  text-align: center;
+  font-weight: 100;
+}
+.divider {
+  border-bottom: 1px solid $light;
+}
+
+
+
+/* Globals for cards */
+
+// Cards
+.cards {
+  background: #f2f2f2;
+  border-radius: 3px;
+  cursor: pointer;
+  height: 17em;
+  padding: 1.5em;
+  position: relative;
+  transition: height 0.4s, box-shadow 0.5s;
+
+  &:hover {
+    box-shadow: 0 0 2em #999;
```

Comment:
This is very bold visual change, not really in phase with the new theme. You'll see soon